src/modules/app/actions/__mocks__/update-env.js

```diff
@@ -0,0 +1,9 @@
+const mockUpdateEnv = jest.genMockFromModule(
```

Comment:
So do we need these __mocks__ are no?

Reply:
I mean as separate files

Reply:
For now, we can merge them in. We can workout how to replace the mock files in another pr.